src/dropdown/dropdownpanel.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import Controller from '../controller.js';
+import ControllerCollection from '/ckeditor5/ui/controllercollection.js';
+
+/**
+ * The basic dropdown panel controller class.
+ *
+ * @memberOf ui.dropdown
+ * @extends ui.Controller
+ */
+
+export default class DropdownPanel extends Controller {
+	/**
+	 * Creates a DropdownPanel instance.
+	 *
+	 * @param {utils.Observable} model
```

Comment:
Missing one param. But since the constructor didn't change, I'd use `@inheritDoc`.